src/screens/create-skill/components/create-skill-form/create-skill-form.tsx

```diff
@@ -0,0 +1,77 @@
+import { Formik } from 'formik';
+import React from 'react';
+import { View } from 'react-native';
+
+import { ButtonMode, HeadingLevel } from 'src/common/enums';
+import { ICreateSkillPayload } from 'src/common/types/skill';
+import { createSkillValidationSchema } from 'src/validation-schemas';
+import { FormInput, FormSelect, Heading, MainButton } from 'src/components';
+
+import {
+  DEFAULT_CREATE_SKILL_PAYLOAD,
+  SKILL_TYPE_OPTIONS,
+} from '../../constants';
+
+import styles from '../../styles';
+
+interface Props {
+  onSubmit: (data: ICreateSkillPayload) => void;
+  onBack: () => void;
+}
+
+const CreateSkillForm: React.FC<Props> = ({ onSubmit, onBack }) => {
+  return (
+    <Formik<ICreateSkillPayload>
```

Comment:
We don't need this type if we set initialValues
```suggestion
    <Formik
```